tests/files/axis/availability_with_ebook_fulfillment.xml

```diff
@@ -1 +1 @@
 and peaches - is generally considered the best in the world.  This is the story of how Darjeeling tea began, was key to the largest tea industry on the globe under Imperial British rule, and came to produce the highest-quality tea leaves anywhere in the world. It is a story rich in history, intrigue and empire, full of adventurers and unlikely successes in culture, mythology and religions, ecology and terroir, all set with a backdrop of the looming Himalayas and drenching monsoons. The story is ripe with the imprint of the Raj as well as the contemporary clout of \xe2\x80\x9cvoodoo farmers\xe2\x80\x9d getting world record prices for their fine teas - and all of it beginning with one of the most audacious acts of corporate smuggling in history.      But it is also the story of how the industry spiraled into decline by the end of the twentieth century, and how this edenic spot in the high Himalayas seethes with union unrest and a violent independence struggle. It is also a front-line fight against the devastating effects of climate change and decades of harming farming practices, a fight that is being fought in some tea gardens - and, astonishingly, won - using radical methods.     Jeff Koehler has written a fascinating chronicle of India and its most sought-after tea. Blending history, politics, and reportage together, along with a collection of recipes that tea-drinkers will love, Darjeeling is an indispensable volume for fans of micro-history and tea fanatics.</annotation><narrator/><edition/><runtime/><fileSize>3.55 MB</fileSize><addedDate>6/25/2015 10:15:01 PM</addedDate><publicationDate>5/12/2015</publicationDate><availability><availableFormats><formatName>Blio</formatName><formatName>ePub</formatName><formatName>AxisNow</formatName></availableFormats><availableCopies>1</availableCopies><totalCopies>2</totalCopies><Checkouts/><Holds/><holdsQueueSize>0</holdsQueueSize><isInHoldQueue>false</isInHoldQueue><estimatedHoldPeriod>0</estimatedHoldPeriod><isReserved>false</isReserved><isCheckedout>true</isCheckedout><checkoutFormat>ePub</checkoutFormat><transactionID>6670197A-D264-447A-86C7-E4CB829C0236</transactionID><downloadUrl>http://adobe.acsm/?src=library&amp;transactionId=2a34598b-12af-41e4-a926-af5e42da7fe5&amp;isbn=9780763654573&amp;format=F2</downloadUrl><checkoutStartDate>6/24/2020 2:34:33 PM +00:00</checkoutStartDate><checkoutEndDate>7/15/2020 2:34:00 PM +00:00</checkoutEndDate><updateDate>6/24/2020 2:40:03 PM +00:00</updateDate></availability><minLoanPeriod>3</minLoanPeriod><maxLoanPeriod>21</maxLoanPeriod><defaultLoanPeriod>21</defaultLoanPeriod><authors><author>Koehler, Jeff</author></authors><narrators/><formatType>EBT</formatType><imageUrl>http://contentcafecloud.baker-taylor.com/Jacket.svc/D65D0665-050A-487B-9908-16E6D8FF5C3E/9781620405147/Medium/Logo</imageUrl><status>OK</status><RTV>true</RTV></title><status><code>0000</code><statusMessage>Availability Data is Successfully retrieved.</statusMessage></status></availabilityResponse>
```

Comment:
I had to update this fixture file slightly. Either the API changed over time, or someone build this example themselves. The download URL here is an ACSM file, but the `checkoutFormat` indicated this is a `Blio` checkout.

This would cause our new code to raise an exception, as a `Blio` checkout format, isn't what we request for Adobe. I validated this against the real API. So the change here (its hard to see because this is all one line) is to update this to `<checkoutFormat>ePub</checkoutFormat>` which is what the API actually responds with.